scripts/build/binutils/binutils.sh

```diff
@@ -176,6 +176,12 @@ do_binutils_backend() {
 
     [ "${CT_TOOLCHAIN_ENABLE_NLS}" != "y" ] && extra_config+=("--disable-nls")
 
+    if [ "${CT_STATIC_TOOLCHAIN}" = "y" ] && [ "${CT_BINUTILS_LINKER_GOLD}" = "y" ]; then
+        # gold links with CXXLINK/g++, not libtool, and does not understand
+        # -all-static
+        extra_config+=("--with-gold-ldflags=--static")
+    fi
+
```

Comment:
I think if we just change the `-all-static` below to `-static` it'll work for both libtool and g++. But the bigger question is if we change things to allow gold to be statically linked and used for linking glibc. There's some old code that stops this combination but I'm not sure any of the justification is still relevant.

Reply:
That could very well be - I have mostly tried the combination glibc 2.27 and binutils 2.34 and cannot recall any problems.

Reply:
Oh, I might have misunderstood this. I also build bfd and keep it the default, ending up with two linkers. There are issues like this https://sourceware.org/bugzilla/show_bug.cgi?id=14995 but I don't know if it still applies.